examples/basics/quickstart/README.md

```diff
@@ -0,0 +1,161 @@
+# Quickstart
+
+This is a simple example showing how you can quickly get started deploying Large Language Models with Dynamo.
+
+## Prerequisites
+
+Before running this example, ensure you have the following services running:
+
+- **etcd**: A distributed key-value store used for service discovery and metadata storage
+- **NATS**: A high-performance message broker for inter-component communication
+
+You can start these services using Docker Compose:
+
+```bash
+docker compose -f deploy/metrics/docker-compose.yml up -d
+```
+
+## Components
+
+- [Frontend](../../../components/frontend/README) - A built-in component that launches an OpenAI compliant HTTP server, a pre-processor, and a router in a single process
+- [vLLM Backend](../../../components/backends/vllm/README) - A built-in component that runs vLLM within the Dynamo runtime
+
+```mermaid
+---
+title: Request Flow
+---
+flowchart TD
+    A["Users/Clients<br/>(HTTP)"] --> B["Frontend<br/>HTTP API endpoint<br/>(OpenAI Style)"]
+    B --> C["NATS Message Broker<br/>(Inter-component communication)"]
+    C --> D["vLLM Backend<br/>(NATS subscriber)"]
+    D --> C
+    C --> B
+    B --> A
+```
+
+## Instructions
+
+There are three steps to deploy and use LLM with Dynamo.
+
+### 1. Launch Engine
+
+**Open a new terminal** and run:
+
+```bash
+python -m dynamo.vllm --model Qwen/Qwen3-0.6B
+```
+
+Leave this terminal running - it will show vLLM Backend logs.
```

Comment:
> it will show vLLM Backend logs

Nice. SDK/serve mixed the logs of all the components, and people were asking to change that, so great to highlight it.